### ISSUE TITLE
Fingerprint Validation on client/VEN end

### ISSUE THREAD

**Issue by muhammadvellani** [OpenLEADR/openleadr-python#56]
I have provided a pair of PEM-formatted certificates and key to VTN and another pair to VEN, according to the docs, all outgoing messages will be cryptographically signed. I tried feeding incorrect fingerprints and no warning or errors are raised. On the client/VEN end, when 
 validate_xml_signature(tree) function is called in client.py to verify the VTN fingerprint provided to the VEN against the fingerprint received

![image](https://user-images.githubusercontent.com/60758355/109055185-af678800-7693-11eb-8c12-5e9b0b2eabc1.png)

but since cert_fingerprint is none, it doesn't go into checking the received message vs the server fed fingerprint hence not raising the error if fingerprint don't match. Is this for a reason? or not implemented yet or am I missing something?

**Comment by stan-janssen** [comment]
Did you provide the `vtn_fingerprint` when creating the client?

```python3

client = OpenADRClient(...
                       vtn_fingerprint='00:11:22:33...')
```

It looks like the client does not know the VTN's fingerprint and can therefore not validate it.

**Comment by stan-janssen** [comment]
Thanks for the report! I looked into it, and you were right; I left the `cert_fingerprint` argument empty when calling `validate_xml_signature` in the client.

It is fixed in the version on the main branch. I also added a testcase that shows it should be working now.

I will release a new version with this fix later today.

**Comment by stan-janssen** [comment]
The fix is now available in v0.5.22.

**Comment by muhammadvellani** [comment]
Works great now. Thanks for the quick update
